src/common/const.go

```diff
@@ -24,10 +24,9 @@ const (
 	LDAPScopeOnelevel   = "2"
 	LDAPScopeSubtree    = "3"
 
-	RoleSystemAdmin  = 1
-	RoleProjectAdmin = 2
-	RoleDeveloper    = 3
-	RoleGuest        = 4
+	RoleProjectAdmin = 1
```

Comment:
They are for internally communication, and has nothing to do with the role_code in DB right?